plugin/storage/es/dependencystore/storage.go

```diff
@@ -60,17 +60,15 @@ func NewDependencyStore(client es.Client, logger *zap.Logger, indexPrefix string
 // WriteDependencies implements dependencystore.Writer#WriteDependencies.
 func (s *DependencyStore) WriteDependencies(ts time.Time, dependencies []model.DependencyLink) error {
 	indexName := indexWithDate(s.indexPrefix, ts)
-	if err := s.createIndex(indexName); err != nil {
```

Comment:
IIRC this should stay here. The index has to be created

Reply:
@pavolloffay  I don't think so - you can simply "write" to the index and it will be created implicitly - check out the SpanWriter / ServiceWriter. The sole reason this was here was to create the index with explicit mappings. But with this PR we have a template that is applied to a new, implicitly created index automagically.

Reply:
You are right sorry my bad.

Reply:
No worries - thanks for looking into my changes to thoroughly.